### DISCUSSION TITLE
PorousFlow deactivate capillarity

### DISCUSSION THREAD

**Discussion by burritotukituki** [idaholab/moose#27062] [Q&A Modules: Porous Flow] [answered]
Hi,

I am running a benchmark in PorousFlow and ModFlow. Looks like in Modflow the basic implementation of unsaturated flow does not considerate capillarity. Is it possible to deactivate capillarity in PorousFlow to compare results?

Cheers!

**Comment by GiudGiud** [comment]
@cpgr

**Reply by cpgr** [reply, marked as answer]
You could set it to zero using PorousFlowCapillaryPressureConst?

**Comment by burritotukituki** [comment]
ups, I guess that works :)
